docs/getting-started/index.md

```diff
@@ -29,7 +29,13 @@ Note that you won't need a cloud account, as we'll be using the `local_file` res
 
 > If you haven’t installed Terramate yet, please refer to the [installation](../installation.md) section for various options.
 
-To begin, create a new directory, and within it, create a file named `terramate.tm.hcl` with the following content:
+To start using Terramate, there are two possible ways of configuring a Terramate project. If you invoke Terramate inside
+a Git repository, Terramate will automatically assume the top-level of your repository as the project root.
+Suppose you use Terramate in a normal directory that isn't a Git repository. In that case, you must configure the project
```

Comment:
```suggestion
If you want to use Terramate in a directory that isn't a git repository, you must configure the project
```

Reply:
accepted - will still write **Git** though as we do it similar in all other pages of the documentation